CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * support for uploading tar to container [#239](https://github.com/softprops/shiplift/pull/239)
 * fix registry authentication to use URL-safe base64 encoding [#245](https://github.com/softprops/shiplift/pull/245)
 * add StopSignal and StopTimeout to ContainerOptionsBuilder [#248](https://github.com/softprops/shiplift/pull/248)
+* add Image(s)::save() and deprecate Image(s)::export() [#257](https://github.com/softprops/shiplift/pull/257)
```

Comment:
I guess this has to be in the changelog section for the next release, not the existing one, right? :laughing: